docker-compose.prometheus.yaml

```diff
@@ -0,0 +1,33 @@
+version: '3.7'
+services:
+  prometheus:
+    image: prom/prometheus
+    ports:
+      - 9090:9090
+    volumes:
+      - ./docker/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml
+  redis:
+    image: redis
+    ports:
+        - 6379:6379
+  web:
+    image: nginx:1.17-alpine
+    ports:
+      - 8083:80
```

Comment:
Did you pick port 8083 for a particular reason?   This isn't a port pattern I'm familiar with.

Reply:
Hm... no particular reason. Just trying to avoid port collisions... I'll change to 8080